build.py

```diff
@@ -5,120 +5,150 @@
 
 
 def last_git_modified(path, n=1):
-    return subprocess.check_output([
-        'git',
-        'log',
-        '-n', str(n),
-        '--pretty=format:%h',
-        path
-    ]).decode('utf-8').split('\n')[-1]
+    return (
```

Comment:
I applied black formatting to the build.py script and made some adjustments for helm 3.  It looks like more than it is.